LibreNMS/Alert/AlertUtil.php

```diff
@@ -62,14 +59,10 @@ public static function getAlertTransports($alert_id)
 
     /**
      * Returns the default transports
-     *
-     * @return array
      */
-    public static function getDefaultAlertTransports()
+    public static function getDefaultAlertTransports(): array
```

Comment:
You can probably return a Collection here, because it implements ArrayAccess.

The only thing that it causes an issue with is when is_array() is called.

Reply:
I used Tinked to understand what would return the old function and do the same, and I thought it was an array: 

>>> DB::select('SELECT transport_id, transport_type, transport_name FROM alert_transports WHERE is_default=true', (array) []);
=> [
     {#4880
       +"transport_id": 1,
       +"transport_type": "browserpush",
       +"transport_name": "Browser notification",
     },
   ]

>>> AlertTransport::where('is_default', true)->select('transport_id', 'transport_type', 'transport_name')->get()->toArray()
=> [
     [
       "transport_id" => 1,
       "transport_type" => "browserpush",
       "transport_name" => "Browser notification",
     ],
   ]

Is it better to use a collection?

Reply:
Try this in tinker:

```php
$transport = AlertTransport::first()
$transport->transport_name
$trasnport['transport_name']
$trasnport['transport_name'] = 'new name'

$transport
```